ros1_ign_bridge/src/parameter_bridge.cpp

```diff
@@ -33,14 +33,34 @@
 
 #include "ros1_ign_bridge/bridge.hpp"
 
+// Direction of bridge.
+enum Direction
+{
+  // Both directions.
+  BIDIRECTIONAL = 0,
+  // Only from IGN to ROS
+  FROM_IGN_TO_ROS = 1,
+  // Only from ROS to IGN
+  FROM_ROS_TO_IGN = 2,
+};
+
 //////////////////////////////////////////////////
 void usage()
 {
   ROS_ERROR_STREAM(
       "Bridge a collection of ROS1 and Ignition Transport topics.\n\n"
-      << "  parameter_bridge <topic@ROS1_type@Ign_type> .. "
+      << "  parameter_bridge <topic@ROS1_type(@,[,])Ign_type> .. "
       << " <topic@ROS1_type@Ign_type>\n\n"
-      << "E.g.: parameter_bridge /chatter@std_msgs/String@ignition.msgs"
+      << "The first @ symbol delimits the topic name from the message types.\n"
+      << "Following the first @ symbol is the ROS1 message type.\n"
+      << "The ROS1 message type is followed by an @, [, or ] symbol where\n"
+      << "    @  ==  a bidirectional bridge, \n"
+      << "    [  == a bridge from Ignition to ROS1,\n"
+      << "    ]  == a bridge from ROS1 to Ignition.\n"
+      << "Following the direction symbol is the Ignition Transport message "
+      << "type.\n"
+      << "A bidirectional bridge example:\n"
+      << "    parameter_bridge /chatter@std_msgs/String@ignition.msgs"
```

Comment:
Maybe add examples of unidirectional here.

Reply:
Added examples for each.